ci/manager.sh

```diff
@@ -45,6 +45,13 @@ else
     REQUEST=$4
 fi
 
+if [[ -z "$5" ]]
+then
+    AUTHORIZED=true
```

Comment:
I think we can make this better. Instead of AUTHORIZED we can pass the level of test that we want to run. 
for example
```
1 -> unauthorized (lint + branch check)
2 -> normal authorized level + speed tester
3 -> Add pktgen
4 -> Add mtcp and stuff
```

Then from the python script auth sequence we will decide which test level to run.